cogs/moderator.py

```diff
@@ -76,7 +77,12 @@ async def verify(self, ctx: Context, member):  # parameter only for pretty help.
         """
         global verified
         try:
-            member: Union[Member, User] = ctx.message.mentions[0]
+            if ctx.message.mentions:
+                member: Union[Member, User] = ctx.message.mentions[0]
+            elif ctx.message.content.split(" ")[1].isnumeric():
+                member: Union[Member, User] = self.bot.get_user(
+                    int(ctx.message.content.split(" ")[1])
+                )
         except IndexError:
```

Comment:
This can be easily gotten by using the `member` attribute of the function. Since no Type is given, discord.py will accept any Type.

Also the `ValueError` exception should be in the catch statement since we don't want any runaway exceptions.

Also you should use the `Guild` Attribute of the `Context` Attribute to get a Member, this is more efficient as not every Guild needs to be checked.

```suggestion
                member: Union[Member, User] = ctx.guild.get_member(int(member))
        except (IndexError, ValueError):
```